packages/nimble-tokens/build/svg-to-ts-config.js

```diff
@@ -10,25 +10,28 @@ const svgToTsConfig = {
     srcFiles: [
         iconAssetDir
     ],
-    conversionType: 'constants',
     prefix: '',
     svgoConfig: {
         // https://github.com/svg/svgo#built-in-plugins
         plugins: [
             {
-                cleanupAttrs: true
-            },
-            {
-                removeUnknownsAndDefaults: {
-                    keepDataAttrs: false
-                }
-            },
+                name: 'preset-default',
```

Comment:
Change in configuration options for svg to ts child dependency svgo.
`cleanupAttrs` is enabled by default in the preset-default so removed the explicit config: https://github.com/svg/svgo#built-in-plugins